desktop-ui/presentation/presentation.cpp

```diff
@@ -245,34 +252,23 @@ auto Presentation::resizeWindow() -> void {
   if(fullScreen()) setFullScreen(false);
   if(maximized()) setMaximized(false);
 
-  u32 multiplier = max(2, settings.video.multiplier);
-  u32 viewportWidth = 320 * multiplier;
-  u32 viewportHeight = 240 * multiplier;
+  u32 scale = settings.video.scale;
+  u32 viewportWidth = 320 * scale;
+  u32 viewportHeight = 240 * scale;
 
   if(emulator && program.screens) {
     auto& node = program.screens.first();
     u32 videoWidth = node->width() * node->scaleX();
     u32 videoHeight = node->height() * node->scaleY();
     if(settings.video.aspectCorrection) videoWidth = videoWidth * node->aspectX() / node->aspectY();
     if(node->rotation() == 90 || node->rotation() == 270) swap(videoWidth, videoHeight);
-
-    u32 multiplierX = viewportWidth / videoWidth;
-    u32 multiplierY = viewportHeight / videoHeight;
-    u32 multiplier = max(1, min(multiplierX, multiplierY));
-
-    viewportWidth = videoWidth * multiplier;
-    viewportHeight = videoHeight * multiplier;
+    viewportWidth = videoWidth * scale;
+    viewportHeight = videoHeight * scale;
   }
 
   u32 statusHeight = showStatusBarSetting.checked() ? StatusHeight : 0;
-
-  if(settings.video.autoCentering) {
-    setGeometry(Alignment::Center, {viewportWidth, viewportHeight + statusHeight});
-  } else {
-    setSize({viewportWidth, viewportHeight + statusHeight});
-  }
-
-  setMinimumSize({256, 240 + statusHeight});
+  setSize({viewportWidth, viewportHeight + statusHeight});
+  setMinimumSize({16, 16 + statusHeight});
```

Comment:
16x16 is a very small minimum window size, are you sure about that value?

Reply:
Yeah I didn't know what to put here, maybe something like 160x144 is better since it's the lowest possible system resolution?

Reply:
The only reason I can think of that this code exists is to prevent the user from resizing the menu away and the not figuring out that they can resize out of it? Because I think Windows itself safeguards the frame from being resized into oblivion. And I'd think any competent OS would.

(fwiw I don't personally like the resize ability, as it results in accidental resizes when trying to drag the title bar and you have to rescale twice to normalize the frame.)